cheddar/cheddar-context/src/test/java/com/clicktravel/cheddar/request/context/DefaultSecurityContextTest.java

```diff
@@ -47,9 +49,11 @@ public void shouldReturnTeamId_withTeamInContext() {
         // Given
         final String userId = randomId();
         final String teamId = randomId();
+        final String identityProviderId = randomId();
```

Comment:
I may just be missing it but is there a test to validate the setting of the identityProviderId via the get identityProviderId() method. A check on that seems to be missing in this test class?

Reply:
Thanks, I have now added a dedicated identityProviderId test. :)